services/web/server/src/simcore_service_webserver/computation_comp_tasks_listening_task.py

```diff
@@ -165,12 +173,40 @@ async def listen(app: web.Application):
                 continue
 
 
+async def listening_channel_pg_engine(app: web.Application) -> Engine:
+    cfg = app[APP_CONFIG_KEY][CONFIG_SECTION_NAME]
+    pg_cfg = cfg["postgres"]
+
+    dsn = DataSourceName(
+        application_name=f"{__name__}_{id(app)}",
+        database=pg_cfg["database"],
+        user=pg_cfg["user"],
+        password=pg_cfg["password"],
+        host=pg_cfg["long_running_session_host"],
+        port=pg_cfg["port"],
+    )
+    MIN_SIZE = 1
+    MAX_SIZE = 1
+    log.info("Creating listening pg engine for %s", dsn)
+    for attempt in Retrying(**PostgresRetryPolicyUponInitialization(log).kwargs):
+        with attempt:
+            engine = await create_pg_engine(dsn, minsize=MIN_SIZE, maxsize=MAX_SIZE)
```

Comment:
shouldn't we make sure there is a SINGLE engine per application? 
- the db app submodule could create them upon startup and is responsible of closing them as well ( e.g. appending to ``app.cleanup_ctx`` )
- See ``setup`` in ``services/web/server/src/simcore_service_webserver/db.py``
- This way, we can monitor the status of all connections to the db, either direct or via the pgbouncer

Reply:
done